### DISCUSSION TITLE
How to move Wikijs server to another machine ?

### DISCUSSION THREAD

**Discussion by Jason-MK** [requarks/wiki#6054] [Help / Questions]
Hello ,

First of all I want to know how exactly should I move my WIKIjs server to another machine I need to move Debian to CentOS 

Do I have to move Database ? and how do I move Wiki files to the new machine ? 

I really need your help !

Best Regards

**Comment by fnafox** [comment]
The following article provides the necessary information:
https://docs.requarks.io/install/transfer

tl;dr:
1. create new instance
2. import old database
3. restart instance

**Reply by Jason-MK** [reply]
> The following article provides the necessary information: https://docs.requarks.io/install/transfer
> 
> tl;dr:
> 
> 1. create new instance
> 2. import old database
> 3. restart instance

But I didn't install it using Docker so can I do it ?
